pkg/authorization/conditions/path.go

```diff
@@ -0,0 +1,32 @@
+package conditions
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/sessions"
+)
+
+type path struct {
+	path *regexp.Regexp
+}
+
+// NewPath builds the PathRegex from a raw path regex string
+func NewPath(pathRegex string) (Condition, error) {
+	compiled, err := regexp.Compile(pathRegex)
+	if err != nil {
+		return nil, fmt.Errorf("error compiling regex /%s/: %v", pathRegex, err)
```

Comment:
What are the slashes for around the %s? Perhaps we should just use %q instead?
```suggestion
		return nil, fmt.Errorf("error compiling regex %q: %v", pathRegex, err)
```

Reply:
It was my intent to have it mirror other languages that traditionally wrapped regexes in `/regex/` by default (perl, sec, etc) as a visual flair to show it was a regex.

Reply:
Gotta agree with Nick on this one

Reply:
Ok, sure, that's reasonable